src/components/Profile/ProfileEdit.js

```diff
@@ -16,17 +16,24 @@ import { Check } from '../styleElements/icons/Check.js';
 import FileUploader from './FileUploader';
 import { Settings } from '../styleElements/icons/Settings';
 import { FilterPage } from '../FilterPage';
+import { useDispatch, useSelector } from 'react-redux';
+import { updateUser } from '../../Redux/userSlice';
+import { selectActiveFilters } from '../../Redux/filtersSlice';
+import { selectCurrentUser } from '../../Redux/userSlice';
 
-const useStyles = makeStyles(() => ({
-  formControl: {
-    minWidth: 100,
-  },
-}));
+// const useStyles = makeStyles(() => ({
+//   formControl: {
+//     minWidth: 100,
+//   },
+// }));
```

Comment:
you can just delete this stuff if it's not needed

Reply:
I was a bit nervous to take a machete to this page since it's the one Filip was working on.  But you're right, I should just do what needs to be done.